CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+The Flux Terraform provider is [Apache 2.0
+licensed](https://github.com/fluxcd/flux2/blob/main/LICENSE) and
+accepts contributions via GitHub pull requests. This document outlines
+some of the conventions on to make it easier to get your contribution
+accepted.
+
+We gratefully welcome improvements to issues and documentation as well as to
+code.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. We require that you sign-off all your commits, see the
+[DCO](DCO) file for details before you start contributing code to FluxCD
+organization.
```

Comment:
```suggestion
By contributing to the Flux project you agree to the Developer Certificate of
Origin (DCO). This document was created by the Linux Kernel community and is a
simple statement that you, as a contributor, have the legal right to make the
contribution. 

We require all commits to be signed. By signing-off with your signature, you
certify that you wrote the patch or otherwise have the right to contribute the 
material by the rules of the DCO:

```
Signed-off-by: Jane Doe <jane.doe@example.com>
```

The signature must contain your real name (sorry, no pseudonyms or anonymous contributions)

If your `user.name` and `user.email` are configured in your Git config, you can sign your commit automatically with `git commit -s`.
```